yang/ietf-te-types-rdnm-ext.yang

```diff
@@ -0,0 +1,68 @@
+module ietf-te-types-rdnm-ext {

```

Comment:
it seems to me the suggestion above by Italo was to avoid "te" : ietf-rdnm-types

Reply:
agree